src/azure-cli-core/azure/cli/core/decorators.py

```diff
@@ -73,7 +73,8 @@ def _wrapped_func(*args, **kwargs):
             try:
                 return func(*args, **kwargs)
             except Exception as ex:  # nopa pylint: disable=broad-except
-                get_logger(__name__).info('Suppress exception %s', ex)
+                import traceback
+                get_logger(__name__).info('Suppress exception:\n%s', traceback.format_exc())
```

Comment:
The old implementation can be easily ignored:

```
> az verion --debug
Suppress exception DLL load failed while importing win32file: The specified module could not be found.
```

With this change, it shows a more informative error:

```
> az verion --debug
Suppress exception:
Traceback (most recent call last):
  File "d:\cli\azure-cli\src\azure-cli\azure\cli\__main__.py", line 60, in <module>
    raise ex
  File "d:\cli\azure-cli\src\azure-cli\azure\cli\__main__.py", line 53, in <module>
    sys.exit(exit_code)
SystemExit: 0

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\cli\azure-cli\src\azure-cli-telemetry\azure\cli\telemetry\__init__.py", line 11, in <module>
    import portalocker
  File "D:\cli\env38\lib\site-packages\portalocker\__init__.py", line 4, in <module>
    from . import portalocker
  File "D:\cli\env38\lib\site-packages\portalocker\portalocker.py", line 9, in <module>
    import win32file
ImportError: DLL load failed while importing win32file: The specified module could not be found.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\cli\azure-cli\src\azure-cli-core\azure\cli\core\decorators.py", line 74, in _wrapped_func
    return func(*args, **kwargs)
  File "d:\cli\azure-cli\src\azure-cli-core\azure\cli\core\telemetry.py", line 248, in conclude
    from azure.cli.telemetry import save
  File "d:\cli\azure-cli\src\azure-cli-telemetry\azure\cli\telemetry\__init__.py", line 24, in <module>
    import portalocker
  File "D:\cli\env38\lib\site-packages\portalocker\__init__.py", line 4, in <module>
    from . import portalocker
  File "D:\cli\env38\lib\site-packages\portalocker\portalocker.py", line 9, in <module>
    import win32file
ImportError: DLL load failed while importing win32file: The specified module could not be found.
```